src/vs/platform/actionWidget/browser/actionWidget.css

```
@@ -105,9 +105,10 @@
 }
 
 .action-widget .monaco-list-row.action .title {
-	flex: 1;
+	/* Don't let action title truncate so we can see the full title */
```

```suggestion
	/* MEMBRANE: Don't let action title truncate so we can see the full title */
```
We use `MEMBRANE` as a prefix to easily identify our changes

---

Ah, got it thanks!